### PR TITLE
Reporting feature yueru

### PR DESCRIPTION
**Pull request by YueruZhao**

### CODE REVIEW

File: src/components/Reports/Reports.jsx [hunk 1]
```diff
@@ -1,17 +1,223 @@
...
+      allTeams =allTeams.filter(team => team.isActive ===false);
+    }
+
+
     return (
       <div>
-        <div className="jumbotron">ReportsPage</div>
         <div>
           <nav className="navbar navbar-expand-md navbar-light bg-light mb-3 nav-fill">
             <li className="navbar-brand">Generate Report:</li>
```
Comment (chrisweilacker) [inline src/components/Reports/Reports.jsx:145]: I think here you can display which category is selected somehow, right now it isn't obvious what is selected.

File: src/components/Reports/Reports.jsx [hunk 2]
```diff
@@ -1,17 +1,223 @@
...
+    this.setActive=this.setActive.bind(this)
+    this.setInActive=this.setInActive.bind(this)
+    this.setAll=this.setAll.bind(this)
+  }
+
+  componentDidMount() {
+    this.props.fetchAllProjects(); // Fetch to get all projects
+    this.props.getAllUserTeams();
+    this.state = {
+      showProjects: false,
```
Comment (chrisweilacker) [inline src/components/Reports/Reports.jsx:58]: I think you can set one of these to true as the 'default' value that will be shown.

File: src/components/Reports/Reports.jsx [hunk 3]
```diff
@@ -1,17 +1,223 @@
...
+          <button style={{margin:'3px'}} exact className="btn btn-secondary btn-bg mt-3" onClick={this.showTeamsTable}>Team</button>
+          <div>
+          <a>Select a Filter</a>
+            <input name='radio' type="radio" style={{margin:'5px'}} value="active" onChange={this.setActive}  />
+            Active
+            <input name='radio' type="radio" style={{margin:'5px'}} value="inactive" onChange={this.setInActive } />
+            InActive
+            <input name='radio' type="radio" style={{margin:'5px'}} value="all" onChange={this.setAll }defaultChecked />
+            All
+          <button style={{margin:'5px'}} type="submit" className="btn btn-primary btn-bg mt-3">
```
Comment (chrisweilacker) [inline src/components/Reports/Reports.jsx:159]: You should try to put the submit button on a  new line you can look at reactstrap examples to get that to work or can just make another div.

File: src/components/Reports/Reports.jsx [hunk 4]
```diff
@@ -1,17 +1,223 @@
...
+
+
     return (
       <div>
-        <div className="jumbotron">ReportsPage</div>
         <div>
           <nav className="navbar navbar-expand-md navbar-light bg-light mb-3 nav-fill">
             <li className="navbar-brand">Generate Report:</li>
           </nav>
+          <a>Select a Category</a>
```
Comment (chrisweilacker) [inline src/components/Reports/Reports.jsx:147]: If you are going to have the table centered this should be centered as well on top of it.

**Review by chrisweilacker — APPROVED** [review]
It is functional, but design/initial state could be improved, I made comments you can try to make the changes and push a new commit up or merge first and then push an commit up later. I leave it up to you to merge the code in.

32d0c15a3f06e6e5ce4841c1e071215

File: .env [hunk 5]
```diff
@@ -0,0 +1,2 @@
+REACT_APP_APIENDPOINT="http://localhost:4500/api"
+SKIP_PREFLIGHT_CHECK=true
```
Comment (getnithesh) [inline .env:2]: please revert this file, we are not supposed to commit the env files as it contains sensitive data.

**Review by getnithesh — COMMENTED** [review]
I think there are lot of editor related files has been committed to this PR, please ignore them by adding the ./idea folder to gitignore

**Review by chrisweilacker — DISMISSED** [review]
Approved

**Review by chrisweilacker — APPROVED** [review]

**Pull request merged**